docs/client.rst

```diff
@@ -0,0 +1,137 @@
+The SecureDrop Client Application
+=================================
+
+As part of the ongoing work to make a user (journalist?) friendly workstation
+for SecureDrop we have created a native client application to be run within the
+Qubes operating system. It helps journalists with the most common activities
+associated with using SecureDrop in a user friendly manner.
+
+Currently the client is alpha quality although work is ongoing in terms of
+improving features and the user interface.
+
+The source code, and related issues are `hosted on GitHub <https://github.com/freedomofpress/securedrop-client>`_.
+
+Developer Setup
+---------------
+
+Clone the repository::
+
+    git clone git@github.com:freedomofpress/securedrop-client.git
+
+Create and enable a new Python3 virtualenv::
```

Comment:
Why is this step here? Are we suggesting they install `pipenv` within a venv? I feel like to avoid confusion we might just suggest they `sudo pip install pipenv` or `pip install --user pipenv`